ui/src/components/connections/ConnectionsTable.tsx

```diff
@@ -29,6 +29,7 @@ import { NoResults } from "src/components/shared/NoResults";
 import { SiderLayoutContent } from "src/components/shared/SiderLayoutContent";
 import { TableCard } from "src/components/shared/TableCard";
```

Comment:
We are getting 2 deprecation errors from this component, can you fix them, please?

<img width="627" alt="image" src="https://github.com/user-attachments/assets/b1eb9d66-7bb5-4ae0-a8f5-3f63de8ffea6">